File: backend/src/main/java/com/example/kanban/repository/TaskRepository.java
package com.example.kanban.repository;

import com.example.kanban.entity.Task;
import com.example.kanban.entity.User;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.List;

public interface TaskRepository extends JpaRepository<Task, Long> {
    List<Task> findAllByUserOrderByOrderAsc(User user);
}

File: backend-kotlin/src/main/kotlin/com/example/kanban/repository/TaskRepository.kt
package com.example.kanban.repository

import com.example.kanban.entity.Task
import com.example.kanban.entity.User
import org.springframework.data.jpa.repository.JpaRepository

interface TaskRepository : JpaRepository<Task, Long> {
    fun findAllByUserOrderByOrderAsc(user: User): List<Task>
}
